### PR TITLE
[45] Product Page using HTML

### PR DESCRIPTION
**Pull request by DamisiM**
## Task: `[45] Product Page using HTML`
- Updated my Product Page using HTML
IMG_20200929_164546
IMG_20200929_164523
IMG_20200929_164538

IMG_20200929_164538

### CODE REVIEW

File: submissions/damisiMahmood/HTML/ecommerceStore-productPage.html [hunk 1]
```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en-US">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="author" content="Damisi Mahmood">
+        <title>I-Damsels Elite Shopping</title>
+    </head>
+    <body>
+        <header><img src="fashion logo.png" alt="All Things Fashion">
+            <h1><span style="font-size: 60px;"> I- DAMSELS ELITE SHOPPING</span></h1>
```
Comment (ToluAkin) [inline submissions/damisiMahmood/HTML/ecommerceStore-productPage.html:10]: @DamisiM Remove the style here

**Review by ToluAkin — CHANGES_REQUESTED** [review]
@DamisiM You are meant to use 2 product images. Adjust the webpage to fit the success criteria and remove the unused images. Also, name your image files with no space between the letters and all named in lowercase.

File: submissions/damisiMahmood/HTML/ecommerceStore-productPage.html (hunk 1 above)
Comment (DamisiM) [reply submissions/damisiMahmood/HTML/ecommerceStore-productPage.html:10]: 🥺😥😥😥

**Review by ToluAkin — CHANGES_REQUESTED** [review]
@DamisiM Rename the image for jimmy choo bag

**Comment by DamisiM** [comment]
It has been renamed, no space, no caps

**Review by ToluAkin — APPROVED** [review]

**Pull request merged**